benchmark/aio/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "aio-benchmark",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "initialize": "yarn clone && yarn setup && yarn update", 
+    "clone": "(git clone https://github.com/angular/angular || true) && cd angular && git fetch origin && git reset --hard dd2a650c3455f3bc0a88f8181758a84aacb25fea",
+    "setup": "cd angular && yarn && cd aio && yarn && yarn setup",
+    "//": "Shouldn't need to install the package twice, but the first install seems to leave two @ngtools/webpack installs around.",
```

Comment:
ouch... can (should) we fix it now?

Reply:
Spent a lot of time trying to work around yarn behaviour when installing local packages and it seems broken in a lot of ways. I've seen double packages being left around, mysterious 'failure to link' and others. Can't say I'm very inclined to continue mucking about in it right now.